ccloud/docs/includes/replicator-overrides.rst

```diff
@@ -0,0 +1,11 @@
+Since the Connect workers are backed to the origin cluster, its embedded producers would write to the origin cluster, which is not desired in this case.
```

Comment:
```suggestion
Since the |kconnect| workers are backed to the origin cluster, its embedded producers would write to the origin cluster, which is not desired in this case.
```